.github/workflows/breaking_change_reminder.yml

```diff
@@ -0,0 +1,42 @@
+name: Workflow for Breaking Change Reminder
+on:
+  pull_request:
+    paths:
+      - 'schema/**'
```

Comment:
Can we add following ones too:
- https://github.com/uber/cadence/blob/master/client/frontend/interface.go

In client repos:
- https://github.com/uber-go/cadence-client/blob/master/client/client.go
- https://github.com/uber/cadence-java-client/blob/master/src/main/java/com/uber/cadence/client/WorkflowStub.java

Good to have:
- https://github.com/uber/cadence/blob/master/client/history/interface.go
- https://github.com/uber/cadence/blob/master/client/matching/interface.go
- https://github.com/uber/cadence/blob/master/client/admin/interface.go (since some admin stuff is automated)

Reply:
Thanks for suggestions. Let me cover those paths as well